lib/server.js

```diff
@@ -53,29 +85,59 @@ export class Http3Server extends Http3WebTransport {
   }
 
   /**
-   * @typedef {object} Http3WTSessionVisitor
-   * @property {'Http3WTSessionVisitor'} purpose
-   * @property {any} object
-   * @property {NativeHttp3WTSession} session
-   * @property {string} path
-   *
-   * @param {Http3WTSessionVisitor} args
+   * @param {Http3WTServerSessionVisitorEvent} args
+   */
+  onHttp3WTSessionVisitor(args) {
+    // create Http3 Visitor
+    if (args.object) {
+      const sesobj = new Http3WTSession({
+        object: args.session,
+        parentobj: this
+      })
+      if (this.sessionController[args.path])
+        this.sessionController[args.path].enqueue(sesobj)
+    } else throw new Error('Http3WTSessionVisitor')
+  }
+
+  /**
```

Comment:
Please merge the following functions into one and use only one event.
Furthermore, it should include an object that may also contain information about the change (i.e. host and address).